jhipster-framework/src/test/java/io/github/jhipster/web/filter/reactive/CachingHttpHeadersFilterTest.java

```diff
@@ -43,8 +41,8 @@ public void cacheHeadersSetWhenPathMatches() {
         WebFilterChain filterChain = (filterExchange) -> {
             try {
                 HttpHeaders headers = filterExchange.getResponse().getHeaders();
-                assertEquals("cache", headers.getPragma());
-                assertEquals("max-age=172800000, public", headers.getCacheControl());
+                assertThat(headers.getPragma()).isEqualTo("cache");
```

Comment:
+1 More assertj!! :)

Reply:
FYI I've also started converting all assertions in code generated by generator-jhipster to assertj (we have a mix of different assertions).